test/integration/iso_test.go

```diff
@@ -89,3 +90,27 @@ func testPackages(t *testing.T) {
 	}
 
 }
+
+func testPersistence(t *testing.T) {
+	minikubeRunner := util.MinikubeRunner{
+		Args:       *args,
+		BinaryPath: *binaryPath,
+		T:          t}
+
+	for _, dir := range []string{
+		"/data",
+		"/var/lib/docker",
+		"/var/lib/cni",
+		"/var/lib/kubelet",
+		"/var/lib/localkube",
+		"/var/lib/rkt",
```

Comment:
Also im not sure if we want to include 

`ln -s /mnt/$PARTNAME/var/lib/rkt-etc /etc/rkt`
in the test as /etc/rkt